### PR TITLE
Adding API doc for compute  + floating IP resources

### PR DESCRIPTION
**Pull request by pdube**
Initial documentation for Instances, Flavors, Images and Floating IPs

### CODE REVIEW

File: source/includes/openstack/_instances.mdown [hunk 1]
```diff
@@ -0,0 +1,137 @@
+### Instances
+
+Manage your compute resources.
```
Comment (simongodard) [inline source/includes/openstack/_instances.mdown]: Not sure I understand this description

Comment (pdube) [inline source/includes/openstack/_instances.mdown]: good point, will change

**Comment by prollynomial** [comment]
Are we using `.mdown` as the extension now? I don't care either way, but it should be consistent. Other than that LGTM.

**Comment by pdube** [comment]
@prollynomial just noticed it is the default that sublime is saving it as. I can change to md

**Pull request merged**